packages/expo-in-app-purchases/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 ### 🐛 Bug fixes
 
 - Made the arguments that were labeled as 'optional' on iOS for `getPurchaseHistoryAsync` and `finishTransactionAsync` actually optional. ([#13904](https://github.com/expo/expo/pull/13904) by [@cruzach](https://github.com/cruzach))
+- `getPurchaseHistoryAsync` no longer accepts a boolean parameter. Instead, it accepts an object containing the key `useGooglePlayCache`. ([#13942](https://github.com/expo/expo/pull/13942) by [@cruzach](https://github.com/cruzach))
+  - `useGooglePlayCache` functions similarly to the the previous `refresh` parameter, except it is clearer in naming. If you had `refresh` as false, you should pass `useGooglePlayCache: true`. If `refresh` was true, you should pass `useGooglePlayCache: false`. Please see the [docs](https://github.com/expo/expo/blob/master/docs/pages/versions/unversioned/sdk/in-app-purchases.md#inapppurchasesgetpurchasehistoryasyncrefresh-boolean) for more information.
```

Comment:
Generally changelog entries should be one-liners — otherwise we can hit such issues:
- expo-ci bot suggests adding links in the wrong place and reformats it (like above)
- VSCode's autoformatter may reindent it back to zero
- our Markdown parser may recognize subsequent lines as separate entries

So, according to [Updating Changelogs](https://github.com/expo/expo/blob/master/guides/contributing/Updating%20Changelogs.md) guide, we should use only text and links in the entry (that means no lists, headers, blockquotes, tables, images, inline HTML).